lib/prompts/promptUtils.ts

```diff
@@ -1,24 +1,45 @@
-// @ts-nocheck
 const inquirer = require('inquirer');
 
-const promptUser = async promptConfig => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const promptUser = async (promptConfig: any) => {
```

Comment:
I think we could probably avoid using `any` here - at minimum could use `object`, but it also might be worth seeing if you can get a typed version of `inquirer` either with a newer version or `@types/inquirer`

Reply:
Ah good call. I made some bigger changes to this in [cc915d2](https://github.com/HubSpot/hubspot-cli/pull/1222/commits/cc915d22819ba914fcdfb84f88f28984e110cb07)

Inquirer has a `inquirer.prompt()` util that does exactly the same thing that this util was doing so I removed it. I also had to bump inquirer to v12 because our version (v8) wasn't an esm and didn't support imports

Reply:
@camden11 scratch that, I ran into a ton of issues with this because inquirer v8 doesn't isn't an esm. I'm going to leave this as is for now because we can actually remove this export completely and directly leverage `inquirer.prompt()` everywhere once we bump the inquirer version to 12. I tried doing this in a way that didn't make the PR too crazy, but there are a ton of tests and files that would need to be updated.